resources/dse-yaml/dse/dse-yaml-dse-transforms.edn

```diff
@@ -31,6 +31,18 @@
                                  :description "Directory to store insights logs. Example value: \"/var/log/cassandra/\""}}
                       :order [:data_dir :log_dir]}
                      :group "Insightful Monitoring"))
+ "6.0.12" (transforms
+           (add-field :role_management_options.stats
+                      {:type "boolean"
+                       :required false})
```

Comment:
I guess the master branch of cass-config-builder expects ":required" fields, right?

Reply:
oops, it does not. I need to update this PR.